engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -62,57 +120,58 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
     *      failed to start
     */
   private def startJob(jobKey: BackendJobDescriptorKey,
-                       inputs: Map[FullyQualifiedName, WdlValue],
+                       symbolCache: SymbolCache,
                        configDescriptor: BackendConfigurationDescriptor,
-                       factory: BackendLifecycleActorFactory): WorkflowExecutionActorState = {
-    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, inputs)
-    val jobExecutionActor = context.actorOf(
-      factory.jobExecutionActorProps(
-        jobDescriptor,
-        BackendConfigurationDescriptor(configDescriptor.backendConfig, configDescriptor.globalConfig)
-      ),
-      s"$workflowId-BackendExecutionActor-${jobDescriptor.key.tag}"
-    )
-    jobExecutionActor ! ExecuteJobCommand
-    WorkflowExecutionInProgressState
-  }
-
-  private def startJob(jobKey: BackendJobDescriptorKey, inputs: Map[FullyQualifiedName, WdlValue]): WorkflowExecutionActorState = {
+                       factory: BackendLifecycleActorFactory): Try[ExecutionStatus] = {
+
+      val call = jobKey.call
+      fetchLocallyQualifiedInputs(jobKey, symbolCache) match {
```

Comment:
That's an open discussion. There are issues with both that we need to figure out. Just passing the whole symbol cache to the backend puts a lot of work on the backend side, all of which I'm not even sure is doable in the context of a Job only. On the other side fetching the outputs (even unveluated) for a call and giving only that to the backend makes it harder in terms of lookup if expressions reference variables in different scopes. 
I'd leave it as is for now and we can have a discussion about it later.